config/composites.yaml

```diff
@@ -0,0 +1,17 @@
+composites:
+  # True: concrete composite (composites are disassembled);
+  # False: virtual composite (composites are written monolithically).
+  # default applies to all composites that are not explicitly mentioned
+  # elsewhere.
+  default: False
+  storageClasses:
+    # Use a dict rather than a list to allow easy merging of user config
+    # into default config. Config class has no way of merging lists.
+    ExposureI: False
+    ExposureF: False
+    ExposureCompositeF: True
+    ExposureCompositeF: True
```

Comment:
Is it possible to set these options for the `Exposure` "base" StorageClass and have it propagate to "derived" StorageClasses?

Reply:
StorageClass doesn't have a hierarchy in python land so that's not possible. We could change that by changing the way we define storage classes to make them explicitly state a parent rather than using YAML substitution.

Reply:
Ok, let's just hope [DM-15238](https://jira.lsstcorp.org/browse/DM-15238) makes this a moot point.

Reply:
Unrelated - there are two `ExposureCompositeF`, should one of them be `ExposureCompositeI`?